### PR TITLE
fix: Fix create event bugs

### LINKED ISSUE #1180: Minor Changes in Create Event Process
### Bug Report    

**Actual Behaviour**
Some minor changes need to be done in the Create Event process.

1) Changing the top bar colour to darker.
2) Step 2 and Step 3 info is not reflected in Edit event.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1180 
Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes:
1) Changed the status bar color as requested by changing the theme.
2) Now the information set in 2nd and 3rd step is visible in edit event process.
3) Checkboxes now show checked state in Edit event mode if they were checked during event creation.
4) Some indentation errors which were over looked earlier have also been resolved.

Screenshots for the change:
screenshot

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventActivity.java [hunk 1]
```diff
@@ -105,6 +107,11 @@ public void onPageScrollStateChanged(int state) {
             if (currentPosition > 0 && currentPosition <= 2)
                 pager.setCurrentItem(currentPosition - 1);
         });
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().clearFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);
+            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary_dark));
+        }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventActivity.java]: This shouldn't be needed

**Comment by sridharjajoo** [comment]
@iamareebjamal Done the required changes and updated the screenshot. But no differentiation can be seen between the status bar and view below.

**Comment by iamareebjamal** [comment]
Why is it so

**Comment by sridharjajoo** [comment]
Actually I had used the wrong style that is why it was happening. I have changed the style used in the manifest now.

**Comment by sridharjajoo** [comment]
@Masquerade0097 @iamareebjamal Kindly review

**Pull request merged**